buildenv/jenkins/omrbuild.groovy

```diff
@@ -0,0 +1,346 @@
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -G Ninja -DOMR_ENV_DATA32=ON -DOMR_DDR=OFF -DOMR_JITBUILDER=OFF -C../cmake/caches/Travis.cmake',
+                'compile' : 'ninja'
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'linux_x86-64' : [
+        'label' : 'Linux && x86',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/',
+            'GTEST_COLOR=0'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -C../cmake/caches/Travis.cmake -DOMR_OPT_CUDA=ON -DOMR_CUDA_HOME=/usr/local/cuda',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'linux_x86-64_cmprssptrs' : [
+        'label' : 'Linux && x86',
+        'environment' : [
+            'PATH+CCACHE=/usr/lib/ccache/',
+            'EXTRA_CONFIGURE_ARGS=--enable-DDR'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'autoconf',
+        'builds' : [
+            [
+                'buildDir' : autoconfBuildDir,
+                'configureArgs': 'SPEC=linux_x86-64_cmprssptrs PLATFORM=amd64-linux64-gcc HAS_AUTOCONF=1 all',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'osx_x86-64' : [
+        'label' : 'OSX && x86',
+        'environment' : [
+            'GTEST_FILTER=-*dump_test_create_dump_*:*NumaSetAffinity:*NumaSetAffinitySuspended:*DeathTest*',
+            'PATH+CCACHE=/usr/local/opt/ccache/libexec',
+            'GTEST_COLOR=0'
+        ],
+        'ccache' : true,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -C../cmake/caches/Travis.cmake',
+                'compile' : defaultCompile
+            ]
+        ],
+        'test' : true,
+        'testArgs' : ''
+    ],
+    'win_x86-64' : [
+        'label' : 'Windows && x86',
+        'environment' : [
+            'GTEST_FILTER=-*dump_test_create_dump_*:*NumaSetAffinity:*NumaSetAffinitySuspended:PortSysinfoTest.sysinfo_test_get_tmp3:ThreadExtendedTest.TestOtherThreadCputime',
+            'GTEST_COLOR=0'
+        ],
+        'ccache' : false,
+        'buildSystem' : 'cmake',
+        'builds' : [
+            [
+                'buildDir' : cmakeBuildDir,
+                'configureArgs' : '-Wdev -G "Visual Studio 11 2012 Win64" -C../cmake/caches/AppVeyor.cmake',
+                'compile' : 'cmake --build . -- /m'
+            ]
+        ],
+        'test' : true,
+        'testArgs' : '-C Debug -j1'
+    ],
+    'zos_390-64' : [
+        'label' : 'zOS && 390',
+        'environment' : [
+            'LIBPATH+EXTRA=/openzdk/jenkins/workspace/PullRequest-zos_390-64/build'
+        ],
```

Comment:
The z/OS failures seen on the PR check [1] are typically because the `LIBPATH` was not properly set and the test was not able to find the location of the libraries it wanted to `dlopen`.

[1] https://ci.eclipse.org/omr/job/zAdam-PullRequest-zos_390-64/6/

Reply:
Good catch. That's what I get for blindly copy-pasting.

Reply:
Appears to be working now. I had to do a bit of a workaround. Explained in the comments.